doc/Getting-Started-Angular.html

```diff
@@ -141,12 +143,12 @@ <h4>Application UI</h4>
 <p>After login to the application, you will 
 see the sample dashboard screen:</p>
 <p>
-<img class="img-thumbnail" alt="Dashboard" height="965" src="images/dashboardV2.png" width="1235" /></p>
+<img class="img-thumbnail" alt="Dashboard" height="965" src="images/dashboardV3.png" width="1235" /></p>
 <h3 id="DocMore">More</h3>
 
 <p>Your solution is up and working.
 See
-<a href="Development-Guide-Angular.html"><span class="text-primary">development guide</span></a> document for more 
+<a href="Getting-Started-Xamarin.html"><span class="text-primary">getting started xamarin</span></a> for xamarin application, <a href="Development-Guide-Angular.html"><span class="text-primary">development guide</span></a> document for more 
```

Comment:
Capital `X` for `Xamarin`? (also: [here](https://github.com/aspnetzero/documents/pull/29/files#diff-4da194f25b0fea35173718c38fbbd369R124))